performance/runner/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{env, fs, io};
+use std::process::{Command, ExitStatus, exit};
+
+
+#[derive(Debug, Clone)]
+struct Metric<'a> {
+    name: &'a str,
+    prepare: &'a str,
```

Comment:
Is `prepare` just like a pre-step that is run before the test starts? It looks like that is the arg name below

Reply:
yeah it's part of hyperfine. there's also cleanup which I can add if it turns out we need that too.